### PR TITLE
Add the http port to query applications by `envid` and `orgid`

### PR DESCRIPTION
**Pull request by flowck**
- Add the endpoint `GET /applications` which is expected to return its results with pagination;
- Other PRs incoming;

### CODE REVIEW

File: api/openapi.yml [hunk 1]
```diff
@@ -48,6 +48,41 @@ paths:
...
         default:
           $ref: '#/components/responses/DefaultError'
+    get:
+      tags:
+        - Applications
+      operationId: getApplications
+      summary: Returns a list of applications based on the org_id and environment_id
+      parameters:
+        - name: environment_id
+          in: header
```
Comment (m110) [inline api/openapi.yml:58]: Wouldn't a `query` be a more usual parameter type here? 🤔

File: api/openapi.yml [hunk 2]
```diff
@@ -297,6 +344,25 @@ components:
           id:
             type: string
 
+    GetApplicationsPayload:
+      required: [total, per_page, current_page, total_pages, data]
+      properties:
+        total:
```
Comment (m110) [inline api/openapi.yml:350]: How about we create a dedicated `pagination` type with the metadata? I guess many endpoint may reuse it. :)

Comment (flowck) [reply api/openapi.yml:350]: @m110 Good idea, how does this model sound?

```json
{
 "pagination": {
    "total": 10,
    "per_page": 2
    "other_props": "more props..."
  },
 "data": [9, 8]
}
```

File: api/openapi.yml (hunk 1 above)
Comment (flowck) [reply api/openapi.yml]: @m110 The beauty of codegen: 61a603d4a14898d8f9f4bcda819dc6e91707cc15

File: api/openapi.yml (hunk 2 above)
Comment (flowck) [reply api/openapi.yml]: @m110  [f06e68a](https://github.com/subscribeddotdev/subscribed-backend/pull/62/commits/f06e68ab23979fa4bafd3f1bc54ef367779b2e38)

**Review by m110 — APPROVED** [review]

**Pull request merged**